pkg/clusterapi/aws/actuator.go

```diff
@@ -507,30 +500,73 @@ write_files:
   owner: 'root:root'
   permissions: '0640'
   content: |
-    openshift_group_type: %[1]s
+    openshift_group_type: {{ .NodeType }}
+{{- if .IsNode }}
 - path: /etc/origin/node/bootstrap.kubeconfig
   owner: 'root:root'
   permissions: '0640'
   encoding: b64
-  content: %[2]s
+  content: {{ .BootstrapKubeconfig }}
+{{- end }}
 runcmd:
 - [ ansible-playbook, /root/openshift_bootstrap/bootstrap.yml]
+{{ if .IsNode -}}
 - [ systemctl, restart, systemd-hostnamed]
 - [ systemctl, restart, NetworkManager]
 - [ systemctl, enable, origin-node]
 - [ systemctl, start, origin-node]
-`
+{{ end }}`
```

Comment:
For consistency, you should strip whitespace from the left for this one, too. If you want to keep the newline that existed before, then move the tick to the next line.